test/bytes.js

```diff
@@ -21,5 +21,7 @@ describe('Test constructor', function(){
   it('Should convert a number into a string with options', function(){
     // This function is testes more accurately in another test suite
     assert.equal(bytes(1000, {thousandsSeparator: ' '}), '1 000B');
+    // Test space before unit option
+    assert.equal(bytes(1024, {spaceBeforeUnit: true}), '1 kB');
```

Comment:
Please add this as an individual test (generally only one assert per `it`) and it should describe what it is testing (that the spaceBeforeUnit works).